docs/amy_database_structure.md

```diff
@@ -0,0 +1,134 @@
+# Primary Tables in AMY
+
+The primary tables used in AMY (that will likely appear in every query) are those that store information on events, persons, memberships, and organizations.  Additional tables provide more information about events, persons, memberships, and organizations.
+
+## Events
+
+`workshops_event` - Primary table for all event data.
+
+### Commonly used fields
+
+* `id` Sequential, automatically assigned integer
+* `start` and `end` Event start and end dates 
+* `slug` Event's unique identifier, in the form YYYY-MM-DD-sitename
+* `url` Event's website. Typically in the format username.github.io/YYYY-MM-DD-sitename
```

Comment:
Now we're supporting any sites, but these from GitHub make it easy to read event's slug.